### PR TITLE
Update readme.md with current instructions for DB creation.

### PR DESCRIPTION
**Pull request by recursion**
Added simple instructions for DB setup with recreateDB script.

### CODE REVIEW

File: client/assets/css/chartist.min.css [hunk 1]
```diff
@@ -0,0 +1 @@
...e;width:100%}.ct-octave:before{display:block;float:left;width:0;height:0;padding-bottom:50%}.ct-octave:after{display:table}.ct-octave>svg{display:block;position:absolute;top:0;left:0}.ct-major-tenth{display:block;position:relative;width:100%}.ct-major-tenth:before{display:block;float:left;width:0;height:0;padding-bottom:40%}.ct-major-tenth:after{display:table}.ct-major-tenth>svg{display:block;position:absolute;top:0;left:0}.ct-major-eleventh{display:block;position:relative;width:100%}.ct-major-eleventh:before{display:block;float:left;width:0;height:0;padding-bottom:37.5%}.ct-major-eleventh:after{display:table}.ct-major-eleventh>svg{display:block;position:absolute;top:0;left:0}.ct-major-twelfth{display:block;position:relative;width:100%}.ct-major-twelfth:before{display:block;float:left;width:0;height:0;padding-bottom:33.3333333333%}.ct-major-twelfth:after{display:table}.ct-major-twelfth>svg{display:block;position:absolute;top:0;left:0}.ct-double-octave{display:block;position:relative;width:100%}.ct-double-octave:before{display:block;float:left;width:0;height:0;padding-bottom:25%}.ct-double-octave:after{display:table}.ct-double-octave>svg{display:block;position:absolute;top:0;left:0}
```
Comment (mnaamani) [inline client/assets/css/chartist.min.css]: I think we should have a common .editorconfig file for having consistent new line at EOF

Comment (recursion) [inline client/assets/css/chartist.min.css]: Yeah totally agree.

Although Im not sure why this comment was appended to the chartist.min.css file?

Was that intentional?

**Comment by mnaamani** [comment]
I'm running into an issue with recreateDB.js script.

Starting with a fresh install of postgres I have these databases 
```
                                List of databases
   Name    |  Owner  | Encoding |   Collate   |    Ctype    |  Access privileges  
-----------+---------+----------+-------------+-------------+---------------------
 mokhtar   | mokhtar | UTF8     | en_US.UTF-8 | en_US.UTF-8 | 
 template0 | mokhtar | UTF8     | en_US.UTF-8 | en_US.UTF-8 | =c/mokhtar         +
           |         |          |             |             | mokhtar=CTc/mokhtar
 template1 | mokhtar | UTF8     | en_US.UTF-8 | en_US.UTF-8 | =c/mokhtar         +
           |         |          |             |             | mokhtar=CTc/mokhtar

```

I dropped all flyptox, flyptox_test, postgres databases and roles.

When I run :
```
Mokhtars-MacBook-Air:FlyptoX mokhtar$ node server/utils/recreateDB.js 
Dropping database!flyptox_test
Granting privileges!
Error when creating database.  { [error: role "postgres" does not exist]
  name: 'error',
  length: 93,
  severity: 'FATAL',
  code: '28000',
  detail: undefined,
  hint: undefined,
  position: undefined,
  internalPosition: undefined,
  internalQuery: undefined,
  where: undefined,
  file: 'miscinit.c',
  line: '362',
  routine: 'InitializeSessionUserId' }
Done!
```

So the script is not creating the database.

**Comment by recursion** [comment]
Did you change your knexfile admin credentials?

**Comment by mnaamani** [comment]
Yeah I noticed that in the readme but to be honest I didn't understand what I had to change it to.
So what is the postgres database used for? Travis.. ?

**Comment by recursion** [comment]
postgres is the default user on some machines, and yes in this case for travis.

the postgres db is never used. When the script runs it needs to be connected to some other db because you cant drop the db you are on. So the script logs into the 'user' db (recursion, or mohktar, or postgres) and from there it drops the flyptox db and recreates it.

**Comment by mnaamani** [comment]
Ok great so I changed the admin section to use mokhtar and mokhtar for DB and role and after dropping all the databases its running fine.

**Comment by recursion** [comment]
Indeed :)

**Comment by mnaamani** [comment]
OK I get it now. But it seems weird to have to connect to one db just to drop other databases... but I'm starting to accept weird things and move along .. merging

**Pull request merged**